dist/images/daemonset.sh

```diff
@@ -33,6 +33,8 @@ OVN_LOGLEVEL_NBCTLD=""
 OVNKUBE_LOGFILE_MAXSIZE=""
 OVNKUBE_LOGFILE_MAXBACKUPS=""
 OVNKUBE_LOGFILE_MAXAGE=""
+OVN_ACL_LOGGING_ENABLE=""
```

Comment:
maybe do the `dist/` changes as a separate commit

Reply:
i'm thinking to keep it as one commit so that it can be easily correlated during the review